docs/manual/installation/contao-installtool.en.md

```diff
@@ -6,6 +6,11 @@ aliases:
 weight: 90
 ---
 
+{{% notice info %}}
+The Contao install tool is neglected as of (Contao 5.x). Corresponding information can be entered directly in the 
```

Comment:
Same as above - "neglected" does not fit at all imho (and is unnecessarily negative).